docs/source/user_guide/napari_plugin.md

```diff
@@ -0,0 +1,161 @@
+play button or by opening the `napari > Preferences` menu and changing
+the `Playback frames per second` setting.
+
+:::{admonition} Video playback limitations
+:class: warning
+
+- During playback you cannot jump at an arbitrary frame by clicking on the
+  slider. Make sure to pause the video first.
+- `napari-video` may struggle to play videos at a high frame rate, depending
+  on your hardware, the video resolution and codec. If you experience
+  performance issues, such as the video freezing or skipping frames,
+  try reducing the playback frames per second or fall back to
+  using a [single image](target-load-frame) as a background.
+:::
+
+
+(target-load-frame)=
+### Load an image
+
+This usually means using a still frame extracted from the video, but in theory
+you could use any image that's in the same coordinate system as the
+tracking data. For example, you could use a schematic diagram of the arena,
+as long as it has the same width and height as the video and is
+properly aligned with the tracking data.
+
+::: {dropdown} Extracting a still frame from a video
+:color: info
+:icon: info
+
+You can extract a still frame using your video player/editor of
+choice. We find the command line tool [`ffmpeg`](https://www.ffmpeg.org/)
+very useful for this task.
+
+To extract the first frame of a video:
+
+```sh
+ffmpeg -i video.mp4 -frames:v 1 first-frame.png
+```
+
+To extract a frame at a specific time stamp (e.g. at 2 seconds):
+
+```sh
+ffmpeg -i video.mp4 -ss 00:00:02 -frames:v 1 frame-2sec.png
+```
+:::
+
+To load any image into `napari`, simply drag and drop the image file into
+the napari window. Alternatively, you can use the `File > Open File(s)` menu
+option and select the file from the file dialog.
+In any case, the image will be loaded as a static
+[image layer](napari:howtos/layers/image.html).
+
+## Load the poses dataset
+
+Now you are ready to load some pose tracks over your chosen background layer.
+
+The `movement` plugin on the right-hand side of the window should contain
+an expanded `Load poses` menu. This contains a `source software` dropdown,
+an `fps`  (frames per second) input field, and a `Browse` button to select
+a file containing predicted poses.
+You may also directly paste a `file path` into the homonymous field.
+
+::: {note}
+See [supported formats](target-supported-formats) for more information on
+the expected software and file formats.
+:::
+
+Once the file path is set, click the `Load` button to load the poses dataset
+into `napari`. The plugin will create a new
+[points layer](napari:howtos/layers/points.html) on top of the existing
+image layer.
+
+You will see a view similar to the one below:
+
+![napari plugin with poses dataset loaded](../_static/napari_plugin_with_poses_as_points.png)
+
+The predicted keypoints are represented as points, colour-coded by
+keypoint ID for single-individual datasets, or by individual ID for
+multi-individual datasets. Hovering with your mouse over a point will
```

Comment:
```suggestion
multi-individual datasets. Hovering with your mouse over a point with the points layer selected will
```
Otherwise the tooltip doesn't work.